docs/getting-started/v3-migration.md

```diff
@@ -105,6 +105,24 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
 * `helpers.getValueAtIndexOrDefault` was renamed to `helpers.valueAtIndexOrDefault`
 * `helpers.easingEffects` was renamed to `helpers.easing.effects`
 * `helpers.log10` was renamed to `helpers.math.log10`
+* `helpers.almostEquals` was renamed to `helpers.math.almostEquals`
+* `helpers.almostWhole` was renamed to `helpers.math.almostWhole`
+* `helpers._setMinAndMax` was renamed to `helpers.math._setMinAndMax`
```

Comment:
don't need to include this and the next one in the migration guide because they're new

Reply:
Good call. I've removed them